lib/generators/paper_trail/install_generator.rb

```diff
@@ -31,6 +32,13 @@ def create_migration_file
       end
     end
 
+    def create_initializer
+      create_file(
```

Comment:
Either that, or remove the deprecation warning, but I'm not sure what ramifications that has for users upgrading.